From 8f494636afb9a9b9c0619b256252d3a9dcdbab72 Mon Sep 17 00:00:00 2001
From: flow <thiagodonato300@gmail.com>
Date: Thu, 21 Apr 2022 06:38:18 -0300
Subject: [PATCH] fix: Build iconfix as static library

Otherwise the launcher is unable to find the iconfix .so, and so the
launcher doesn't run.

Signed-off-by: Thiago Donato Ferreira <flowlnlnln@gmail.com>
---
 libraries/iconfix/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libraries/iconfix/CMakeLists.txt b/libraries/iconfix/CMakeLists.txt
index 08441203..97a59129 100644
--- a/libraries/iconfix/CMakeLists.txt
+++ b/libraries/iconfix/CMakeLists.txt
@@ -12,7 +12,7 @@ internal/qiconloader.cpp
 internal/qiconloader_p.h
 )
 
-add_library(Launcher_iconfix ${ICONFIX_SOURCES})
+add_library(Launcher_iconfix STATIC ${ICONFIX_SOURCES})
 target_include_directories(Launcher_iconfix PUBLIC ${CMAKE_CURRENT_SOURCE_DIR} "${CMAKE_CURRENT_BINARY_DIR}" )
 
 target_link_libraries(Launcher_iconfix Qt5::Core Qt5::Widgets)
-- 
2.35.1

